cdc/kv/region_worker.go

```diff
@@ -228,17 +228,12 @@ func (w *regionWorker) checkRegionStateEmpty() (empty bool) {
 
 // checkShouldExit checks whether the region worker should exit, if should exit
 // return an error
-func (w *regionWorker) checkShouldExit(addr string) error {
+func (w *regionWorker) checkShouldExit() error {
 	empty := w.checkRegionStateEmpty()
 	// If there is not region maintained by this region worker, exit it and
 	// cancel the gRPC stream.
 	if empty {
-		cancel, ok := w.session.getStreamCancel(addr)
-		if ok {
-			cancel()
-		} else {
-			log.Warn("gRPC stream cancel func not found", zap.String("addr", addr))
-		}
+		w.cancelStream(0)
```

Comment:
```suggestion
		w.cancelStream(time.Duration(0))
```

Add type, it improves readability.